io.mrarm.mcpelauncher-ui.metainfo.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>io.mrarm.mcpelauncher-ui</id>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>GPL-3.0</project_license>
+    <name>Minecraft Bedrock Launcher UI</name>
+    <summary>Download and play Minecraft Bedrock Edition on GNU/Linux</summary>
+
+    <description>
+        <p>
+            You have to install io.mrarm.mcpelauncher-client for the game launcher and io.mrarm.msa-daemon for Xbox live.
+        </p>
+        <p>
+            The project runs the native libraries from the Android version directly on your computer. This is accomplished by fixing the incompatibilities between the libc used on Android and the one used on desktop Linux or OS X (Android - Bionic; Linux - glibc). This is a simple compatibility layer which doesn’t impact performance in any significant way; it’s similar to Wine, but much more lightweight and simple. Aditionally, all Android-specific code has been rewritten to run on Linux (AppPlatform, Store, Xbox Live, etc.).
+        </p>
+    </description>
+
+    <launchable type="desktop-id">io.mrarm.mcpelauncher-ui.desktop</launchable>
+
+    <screenshots>
+        <screenshot type="default">
+        <caption>Main screen of the Launcher UI</caption>
+        <image>https://user-images.githubusercontent.com/24195572/71060819-64f78c80-211b-11ea-8ea4-f1c2dabb9b54.png</image>
+        </screenshot>
+    </screenshots>
+    <categories>
+        <category>Game</category>
+    </categories>
+
+    <recommends>
+        <id>io.mrarm.mcpelauncher-client</id>
+        <id>io.mrarm.msa-daemon</id>
+    </recommends>
+
+    <content_rating type="oars-1.1" />
```

Comment:
Passable for a launcher but it should describe Minecraft's rating.

Reply:
Added a more detailed rating.
Althought the game has ingame purchases on google play, you cannot buy the ingame coins with this launcher.